src/arch/xtensa/configs/override/cannonlake_chrome_override.config

```diff
@@ -0,0 +1,2 @@
+CONFIG_CAVS_LPS=y
```

Comment:
I think that `override` in file name is not needed, cos it's already in `override` folder.
@lbetlej To make things easier I can add automatic generation of targets for overrides, just like for defconfigs.
So instead of something like:
```
make cannonlake_defconfig
cp ../src/arch/xtensa/configs/override/cannonlake_chrome.config .override
make overrideconfig
```
There will be
```
make cannonlake_defconfig
make cannonlake_chrome_override_config
```

Reply:
BTW I'm not sure if these files should have `.config` suffixes. I know defconfigs don't have extension in kernel, but now sure about non-defconfigs.
@lgirdwood Can you tell us, what's the best  practice here?

Reply:
@jajanusz I like the idea , Could be multiple stages supported ? Like:

```
make cannonlake_defconfig
make cannonlake_chrome_override_config
make cannonlaje_debug
```

Reply:
Yes you can stack them.

Reply:
@kv2019i any comment on stacking the kconfigs ?

Reply:
> BTW I'm not sure if these files should have `.config` suffixes. I know defconfigs don't have extension in kernel, but not sure about non-defconfigs.
> @lgirdwood Can you tell us, what's the best practice here?

@kv2019i Can  you share the best practice?

Reply:
@jajanusz @lgirdwood Stacking is very common. Typically you merge the stacked configs to a single .config file, before running the actual build make target. E.g. for SOF kernel we have following instructions:
https://github.com/thesofproject/kconfig/blob/master/README.md 

The filename conventions vary but e.g. in kernel non-defconfig files still have the ".config" suffix (e.g. arch/x86/Kconfig.debug). But e.g. in SOF kconfig repo, all config files have a "defconfig" suffix, so this is not universally followed (upstream kernel has exceptions as well).

Reply:
ok thanks for info, we will go for .config then for non-defconfig files